app/jobs/get_usps_proofing_results_job.rb

```diff
@@ -444,6 +447,16 @@ def mail_delivery_params(proofed_at)
     return { wait_until: wait_until, queue: :intentionally_delayed }
   end
 
+  def send_enrollment_status_sms_notification(enrollment:)
+    return if IdentityConfig.store.in_person_proofing_enabled.blank? ||
```

Comment:
just wondering - why `blank?` instead of `-- false` or `!Identity...` ?

Reply:
also, i think it might be safe not to do this check here: `IdentityConfig.store.in_person_proofing_enabled.blank?`.  We check for this value on line 17 in the perform function and return if it's false (`ipp_enabled?`). So if it is false we wouldn't get to `check_enrollments` which is eventually invoking the new method you're adding.

Reply:
> just wondering - why `blank?` instead of `-- false` or `!Identity...` ?

copied from another job, changed it.

Reply:
> also, i think it might be safe not to do this check here: `IdentityConfig.store.in_person_proofing_enabled.blank?`. We check for this value on line 17 in the perform function and return if it's false (`ipp_enabled?`). So if it is false we wouldn't get to `check_enrollments` which is eventually invoking the new method you're adding.

Good catch, removed it.